### ISSUE TITLE
Add documentation for cwd and from API

### ISSUE THREAD

**Issue by fraxken** [NodeSecure/scanner#21]
Add a root `./docs` folder with complete documentation on how work cwd and from API.

**Comment by mbalabash** [comment]
Hey @fraxken!
Is this issue still vacant? Can I start working on it?

**Comment by fraxken** [comment]
Hello @mbalabash,

Yes i'm not working on this recently (lacking time). You'r welcome on the subject 👍

**Comment by mbalabash** [comment]
@fraxken OK. It will take some time because I am only familiar with `from` use-case.

Few questions to align our expected result:
1. Do you want to describe in docs all `cwd, from, verify` methods?
2. Do you want to include examples of usage for each method in addition to short description of use-case?

**Comment by fraxken** [comment]
cwd and from are the most complicated. My goal with those docs is to bring context and value to contributors (as the project became important for a lot of other of our tools.. drawing the big picture is getting important).

**Comment by fraxken** [comment]
An example of little schema is was working on:

![Scanner (from command) drawio](https://user-images.githubusercontent.com/4438263/155023987-d6dbe1ec-64fa-4582-bceb-9d360d6d9482.png)

I was in the idea of documenting each steps (maybe this could bring you some ideas).

**Comment by mbalabash** [comment]
Maybe I am wrong person for this task 🙃
I am bad at schemes and I don’t know about use-cases which you have in other tools.

**Comment by tekeuange23** [comment]
Question:
Does the doc .md files (/docs/cwd.md and /docs/from.md)

**Comment by fraxken** [comment]
@tekeuange23 Yes with markdown. Already started on my api-documentation branch but I never had the time to finish: https://github.com/NodeSecure/scanner/blob/api-documentation/docs/from.md

**Comment by tekeuange23** [comment]
Cool, I've an awesome template to go
I'm just going to continue on the same branch
